docs/topics/logging.txt

```diff
@@ -299,10 +299,12 @@ Finally, here's an example of a fairly complex logging setup::
         'disable_existing_loggers': False,
         'formatters': {
             'verbose': {
-                'format': '%(levelname)s %(asctime)s %(module)s %(process)d %(thread)d %(message)s'
+                'format': '{levelname} {asctime} {module} {process} {thread} {message}'
```

Comment:
You should use `{process:d}` and `{thread:d}` here otherwise you are changing the behaviour.

Reply:
Also add a trailing comma.

Reply:
Updated.